src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -52,7 +53,7 @@ const Search = ({route, onPress}) => {
 const styles = StyleSheet.create({
   container: {
     flexDirection: 'row',
-    backgroundColor: COLORS.white,
+    backgroundColor: 'white',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been changed to a hardcoded value. It's better to use color constants for consistency and ease of updates.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: 'white',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->